lib/constants.go

```diff
@@ -48,14 +47,44 @@ func EndTimer(timer *monitor.TimeMeasure) {
 	}
 }
 
+// WaitGroupWithError is like a sync.WaitGroup, with an error channel
+type WaitGroupWithError struct {
+	waiter  chan error
+	counter uint
```

Comment:
Using an `uint` here is programmatically correct. But then you have to do things like `uint(len(...))`, which bothers me more than it helps for catching programming errors.
If you want to use `uint` here, then put an `int` in the `StartParallelize`, and panic if the int is < 0. That's easier to use...

Reply:
By the golang best practice, [`panic` should not be used expect in assert-like cases](https://golang.org/doc/effective_go.html#panic), and that masks errors in a onet multiserver setup, making the tests hang and such. I'm trying to get rid of theses in the code bases I'm coming across.
I'm meeting you halfway, adding a `StartParallelizeWithInt` to allow for your usecase.